src/strategies/total-axion-shares/examples.json

```diff
@@ -4,7 +4,8 @@
     "strategy": {
       "name": "total-axion-shares",
       "params": {
-        "symbol": "TAS"
+        "symbol": "TAS",
+        "dataReader": "0x8458fe26CbF3B8295755654C02ABaDFB1d3CBCe6"
       }
     },
     "network": "1",
```

Comment:
In that case can you change the network here plz :)
also make sure to run 
```
yarn test --strategy=total-axion-shares
```
and make sure it pass all the test cases

Reply:
Or else better to not have any changes in examples.json, so it can pass based on previous values